### PR TITLE
Users/v nitja/azsknitinj

### PR DESCRIPTION
**Pull request by v-nitja**
Changes for OMS to change multiple calls to single call.

### CODE REVIEW

File: src/AzSK/Framework/Listeners/OMS/OMSOutput.ps1 [hunk 1]
```diff
@@ -108,10 +108,11 @@ class OMSOutput: ListenerBase
 			{
 				$invocationContext = [System.Management.Automation.InvocationInfo] $currentInstance.InvocationContext
 				$SVTEventContexts = [SVTEventContext[]] $Event.SourceArgs
-				foreach($svtEventContext in $SVTEventContexts)
```
Comment (sbyna-msft) [inline src/AzSK/Framework/Listeners/OMS/OMSOutput.ps1]: Do we need the above registration? like command completed? please check once

File: src/AzSK/Framework/Listeners/OMS/OMSOutput.ps1 [hunk 2]
```diff
@@ -137,41 +138,51 @@ class OMSOutput: ListenerBase
...
-							$this.PublishException($_);
-							[OMSOutput]::IsIssueLogged = $true
+            try{
+                
+				if((-not [string]::IsNullOrWhiteSpace($settings.OMSWorkspaceId)) -or (-not [string]::IsNullOrWhiteSpace($settings.AltOMSWorkspaceId)))
+				{
+					$eventContextAll | ForEach-Object{
+					$eventContext = $_
+						$tempBodyObjects = [OMSHelper]::GetOMSBodyObjects($eventContext,$this.GetAzSKContextDetails()) #need to prioritize this
+
```
Comment (sbyna-msft) [inline src/AzSK/Framework/Listeners/OMS/OMSOutput.ps1]: not clear on the comment

File: src/AzSK/Framework/Listeners/OMS/OMSOutput.ps1 (hunk 1 above)
Comment (v-nitja) [reply src/AzSK/Framework/Listeners/OMS/OMSOutput.ps1]: This feature was implemented in last sprint to capture basic command events in OMS.To Understand if command started from CA got completed or not.

File: src/AzSK/Framework/Listeners/EventHub/EventHubOutput.ps1 [hunk 3]
```diff
@@ -39,30 +39,38 @@ class EventHubOutput: ListenerBase
 		});
 	}
 
-	hidden [void] WriteControlResult([SVTEventContext] $eventContext)
+	hidden [void] WriteControlResult([SVTEventContext[]] $eventContextAll)
 	{
 		try
 		{
 			$settings = [ConfigurationManager]::GetAzSKSettings()
```
Comment (VishalHaibatpure) [inline src/AzSK/Framework/Listeners/EventHub/EventHubOutput.ps1]: EventHub details can be fetched from CA configurations also. Refer OMS file.

Comment (v-nitja) [reply src/AzSK/Framework/Listeners/EventHub/EventHubOutput.ps1]: As discussed with Byna , we can track the same in other user story. Once confirmation on the priority of the same will implement it.

**Pull request merged**